### DISCUSSION TITLE
Wrong link in Vercel docs

### DISCUSSION THREAD

**Discussion by wsuwt** [vercel/community#2954] [Bug Report]
### Page to Investigate

https://vercel.com/docs/concepts/analytics/quickstart

### Steps to Reproduce

On the last steps, '[Deploy your app to Vercel](https://vercel.com/docs/concepts/analytics/quickstart#deploy-your-app-to-vercel)'  you'll see the link [privacy and data compliance standards](https://vercel.com/docs/concepts/analytics/privacy)

It points to Speed Insights policy instead of Privacy policy of Web analytics, https://vercel.com/docs/concepts/analytics/privacy-policy

**Comment by leerob** [comment]
Thank you for reporting this! We'll get this fixed up.
